src/Uno.UWP/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: InternalsVisibleTo("Uno.UI.Toolkit")]
 
 #if __IOS__
-[assembly: Foundation.LinkerSafe]
+[assembly: AssemblyMetadata("IsTrimmable", "True")]
```

Comment:
This may not work with earlier versions of Xamarin.

Reply:
Not sure how to make it conditional though - in VS 16.10.1 it failed build for me because the deprecated attribute was used 🤔